src/daemon.py

```diff
@@ -9,43 +9,92 @@
 from sqlalchemy import text
 from sqlalchemy.engine import Engine
 from src.imbalances_script import RawTokenImbalances
+from src.helper_functions import (
+    get_finalized_block_number,
+)
 from src.config import (
+    initialize_connections,
     CHAIN_SLEEP_TIME,
-    NODE_URL,
-    create_db_connection,
     check_db_connection,
     logger,
 )
+from src.coingecko_pricing import get_price
 
 
-def get_web3_instance() -> Web3:
-    """
-    returns a Web3 instance for the given blockchain via chain name.
-    """
-    return Web3(Web3.HTTPProvider(NODE_URL))
+def read_sql_file(file_path: str) -> str:
```

Comment:
how do you separate what goes to the helper functions and what stays in the main daemon file? I wouldn't mind pushing all short and "independent" functions to the helper functions file so that the main daemon file is more compact.

Reply:
Ok, I can move read_sql_file() to helpers.